docs/computing/mpip.md

```diff
@@ -0,0 +1,198 @@
+# mpiP: Lightweight MPI Profiler
+
+[mpiP](http://mpip.sourceforge.net/) is a lightweight MPI profiler which uses statistical sampling to record profiling data. At the momenmt is only available on Mahti. It generates less overhead and much less data than tracing tools. In order to use no code changes are require, only a re-link is needed. For both Mahti and Puhti, in addition to the required modules, one has the load the mpiP module: 
```

Comment:
The module is available both on mahti and puhti?

Reply:
Hello, 

Yes. The module is available on both Puhti and Mahti.  I added the availability.